.vscode/settings.json

```diff
@@ -1,8 +1,6 @@
 {
   "dart.runPubGetOnPubspecChanges": "always",
-  "dart.analysisExcludedFolders": [
-    "templates"
-  ],
+  "dart.analysisExcludedFolders": ["templates"],
```

Comment:
I think because I added a line in the beginning of this PR to limit the char length and because of that, this file kept getting formatted. Removed it now and should be good.